src/applications/claims-status/components/StemDeniedDetails.jsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
+import moment from 'moment';
+
+const StemDeniedDetails = props => {
```

Comment:
I'd only use the props you need and pass those in explicitly

```suggestion
const StemDeniedDetails = ({deniedAt, remainingEntitlement}) => {
```